asset-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Asset Store
+
+## Overview
+
+This example illustrates how to use the [Asset Store](https://kyma-project.io/docs/1.5/components/asset-store/) to store static webpages.
+
+By default, [Minio](https://min.io/) stores all resources on a cluster, but it also allows you to use different cloud providers. Read the Asset Store [tutorials](https://kyma-project.io/docs/components/asset-store#tutorials-tutorials) for more information.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kyma](https://kyma-project.io/docs/) as the target deployment environment
+
+## Installation
+
+1. Export a GitHub webpage URL of a ready-to-use webpage.
+
+    Example:
+
+    ```bash
+    export GH_WEBPAGE_URL=https://github.com/pPrecel/simple-page-for-asset-store/archive/master.zip
```

Comment:
Can you attach the sample page to this example, instead of your repository? You can filter the content of package in Asset CR.